core/dbt/include/global_project/macros/materializations/models/view/create_or_replace_view.sql

```diff
@@ -34,6 +34,7 @@
     {{ get_create_view_as_sql(target_relation, sql) }}
   {%- endcall %}
 
+  {% do apply_grants(target_relation, grant_config, should_revoke=True) %}
```

Comment:
Similar logic to the default `view` materialization for this:
```suggestion
  {% set should_revoke = do_we_need_to_show_and_revoke_grants(exists_as_view, full_refresh_mode=True) %}
  {% do apply_grants(target_relation, grant_config, should_revoke=should_revoke) %}
```

Reply:
done